httpchecker.go

```diff
@@ -54,6 +54,23 @@ type HTTPChecker struct {
 	// requests. If not set, DefaultHTTPClient is
 	// used.
 	Client *http.Client `json:"-"`
+
+	// Headers is a slice of Headerthat contains the
+	// name and value for the header.
+	// Default is no additional headers are sent.
+	Headers []Header `json:"headers,omitempty"`
```

Comment:
That was my initial thought and implementation. But went the route I did  purely for how it would look in the config (of if using it as a lib). I am happy to change it to http.Header 

I find 

```golang
Headers: []checkup.Header{
  {Name: "Content-Type", Value: "application/json"},
}
```
a little more user friendly than
```golang
Headers: http.Header{
    "Content-Type": []string{ 
       "application/json"
     }  ,
 },
```